config/jobs/kubernetes/kops/kops-periodics-misc.yaml

```diff
@@ -97,6 +97,7 @@ periodics:
     preset-do-spaces-credential: "true"
     preset-do-ssh: "true"
     preset-dind-enabled: "true"
+  cluster: k8s-infra-prow-build-trusted
```

Comment:
```suggestion
  cluster: k8s-infra-prow-build
```

Reply:
👍🏻 updated, and updated all jobs to use the same.